libtransmission/CMakeLists.txt

```diff
@@ -59,13 +59,13 @@ target_sources(${TR_NAME}
         file-win32.cc
         file.cc
         file.h
-        global-ip-cache.cc
-        global-ip-cache.h
         handshake.cc
         handshake.h
         history.h
         inout.cc
         inout.h
+        ip-cache.cc
+        ip-cache.h
```

Comment:
renamed files need a change in xcodeproj, so removing my approval for now

Reply:
Attaching my xcodeproj patch:
[6927.patch](https://github.com/user-attachments/files/16183741/6927.patch)

Reply:
dbf193ee0e77a8c6e31d5c13c45d32f4267f608b Thanks.